db/migrate/20231108093052_create_posts.rb

```diff
@@ -0,0 +1,15 @@
+class CreatePosts < ActiveRecord::Migration[7.1]
+  def change
+    create_table :posts do |t|
+      t.string :title, null: false
+      t.text :content, null: false
```

Comment:
- Hey, the text ` t.text :text` attribute is a `text` and not content. please, kindly update this to be consistent following the erd diagram.